### DISCUSSION TITLE
Multizone Discrete Adjoint Turbomachinery Simulations

### DISCUSSION THREAD

**Discussion by Tushma** [su2code/SU2#1680] [Q&A]
Hello,

I'm running the APU test case and extending it to follow the optimisation procedure described in this paper:

https://arc.aiaa.org/doi/pdf/10.2514/1.B37685

I'm using the FADO scripts to run my optimization but run into an issue when solving the adjoint problem for multizone problems. The solver out puts an error "The required solver doesn't support multizone simulations". How do I solve the adjoint problem for multirow turbomachinery as described in the above paper if the solver does not support multizone simulations. Looking at the SU2_CFD.cpp file it would appear this error appears because I am using discrete adjoint and turbomachinery methods, if this is the case how can SU2 be used for multi row optimisation as described in the literature?

Thank you in advance for any help with this.

**Comment by pcarruscag** [comment]
Hi,
My guess is that this exists in a branch that was not yet merged into master. CC @matteopini

**Reply by Tushma** [reply]
Yes I believe that's what has happened, reverting to an older version in that case, thank you for your help!

**Comment by z-g-h** [comment]
Hello,
Have you now successfully run the APU test case in this paper? I also want to follow the optimisation procedure described in this paper, but I do not know how to write su2 configure file about FFD. Could you show your su2 configure file about it?
Thanks very much.
